lib/msal-node/src/utils/NetworkUtils.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { INetworkModule } from '@azure/msal-common';
+import { HttpClient } from './../network/HttpClient';
+
+export class NetworkUtils {
+    /**
+     * Returns best compatible network client object.
+     */
+    static getNetworkClient(): INetworkModule {
+        return new HttpClient();
```

Comment:
Do we need this util? It was added in browser because we have a possibility of fetch not being available so we fall back to Xhr. Are we going to have multiple network clients in node?

Reply:
Yes -  we'd like to let customers inject their own http client if they choose to.